rs/bitcoin/kyt/tests/tests.rs

```diff
@@ -86,22 +133,60 @@ fn test_check_address() {
     assert!(result.is_err_and(|err| format!("{:?}", err).contains("Not a bitcoin mainnet address")));
 }
 
-#[test]
-fn test_get_inputs() {
-    let (kyt, env) = setup_env();
-    let sender = Principal::anonymous();
-
-    let call_id = env
-        .submit_call(
-            kyt,
-            sender,
-            "get_inputs",
-            Encode!(
-                &"c80763842edc9a697a2114517cf0c138c5403a761ef63cfad1fa6993fa3475ed".to_string()
-            )
-            .unwrap(),
+fn universal_canister_payload(
+    receiver: &PrincipalId,
+    method: &str,
+    payload: Vec<u8>,
+    cycles: Cycles,
+) -> Vec<u8> {
+    wasm()
+        .call_with_cycles(
+            receiver,
+            method,
+            call_args()
+                .other_side(payload)
+                .on_reject(wasm().reject_message().reject()),
+            cycles,
         )
-        .expect("submit_call failed to return call id");
+        .build()
+}
+
+fn submit_universal_call(
+    setup: &Setup,
+    receiver: Principal,
+    method: &str,
+    args: Vec<u8>,
+    cycles: u128,
+) -> Result<RawMessageId, UserError> {
+    let payload =
+        universal_canister_payload(&PrincipalId(receiver), method, args, Cycles::new(cycles));
+    setup.env.submit_call(
+        setup.universal_canister,
+        setup.controller,
+        "update",
+        payload,
+    )
+}
```

Comment:
nit: How about moving those 2 methods `universal_canister_payload` and `submit_universal_call` to an impl block of `Setup`? This would simplify the signature (less parameters).

Reply:
Good suggestions! Done.